sass/_components.scss

```diff
@@ -3385,4 +3387,13 @@
             font-size: $size-3;
         }
     }
+
+    &.is-bottom {
+        width: 100%;
+        position: fixed;
+        bottom: 0;
```

Comment:
Probably going to need to add additional space to the bottom of the page if the countdown banner is stuck to the bottom - otherwise you'll never be able to see the footer (or possibly actual content). Not sure how you're going to manage that though - will probably need to add a class to the body in this scenario.

Reply:
I've added an `if` block in the component template that will add bottom padding to the site content if the Banner is set to be fixed at the bottom. It might need some adjustment though